ios/iosApp/AppInitializer.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import Kingfisher
+import Shared
+import Trikot
+
+enum AppInitializer {
+    static func initializeComponents(environment: AppEnvironment) {
+        initializeCommon()
+        initializeKingfisher()
+    }
+
+    private static func initializeCommon() {
+        TrikotKword.shared.setCurrentLanguage(Foundation.Locale.isPreferredLanguagesFrench ? "fr" : "en")
```

Comment:
This should be done in the common code using expect-actual implementations providing the current language code.